src/feature/redux/books/bookSlice.js

```diff
@@ -1,40 +1,55 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const getBook = createAsyncThunk('book/getBook', async () => {
+  const res = await axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/6zvRNcEtP78mnXzUiDL7/books');
+  return res.data;
+});
```

Comment:
- [ ] Great job fetching data from the API. However, On the home page of your web app books stored in your API aren't displaying, and upon going through your code carefully I noticed you're not dispatching the `getBooks` you have created here. To fix this issue, kindly consider making use of `useEffect` to dispatch `getBooks` this way the books in your `API` will be fetched on page load.

- [ ] Also, make sure your remove book functionalities are working as expected correctly, due to the current bugs in your code, I was unable to test the remove functionalities but it would be nice if you can take a second look at it to make sure everything is working before resubmitting for review.

Happy coding! 😊🚀✅